pass-deposit-services/deposit-core/src/test/java/org/eclipse/pass/deposit/service/SubmissionStatusUpdaterIT.java

```diff
@@ -123,6 +127,7 @@ void testDoUpdate_Success_NoUpdateNotSubmitted() throws IOException {
         // GIVEN
         submission.setSubmissionStatus(SubmissionStatus.SUBMITTED);
         submission.setSubmitted(false);
+        submission.setSubmittedDate(ZonedDateTime.now());
```

Comment:
Probably should have a test that verifies that if a submission.submitteddate is past the status update window, then the submission is not processed for status update.

Reply:
Good point. Will do.